src/web/handlers/FlingGestureHandler.ts

```diff
@@ -7,7 +7,7 @@ import Vector from '../tools/Vector';
 
 const DEFAULT_MAX_DURATION_MS = 800;
 const DEFAULT_MIN_VELOCITY = 700;
-const DEFAULT_MIN_DIRECTION_ALIGNMENT = 0.75;
+const DEFAULT_DIRECTIONAL_ALIGNMENT_CONE = 40;
```

Comment:
Shouldn't this be 45 if we want to cover the full 90 degrees when adjacent directions are set? I think now it leaves two 5-degree cones where the gesture won't be recognized.

Reply:
Since we implemented the diagonal fill, this value can go as low as 30 degrees without leaving blank spots, as 2 cones protrude axially, and then a third one is added between them. Additionally if we decide to leave the division between Diagonal and Axial directions, we already have a function ready to adapt to the missing unfilled space.
In summary, in this particular commit there is a 10 degree cone left blank by the Axial directions, and filled by the Diagonal directions.

Reply:
I ended up using the adaptive method i mentioned.